python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1246,14 +1244,30 @@ def test_parquet_writer_int96_timestamps(tmpdir, pdf, gdf):
     assert_eq(expect, got, check_categorical=False)
 
 
-def test_multifile_warning(datadir):
-    fpath = datadir.__fspath__() + "/*.parquet"
-    with pytest.warns(UserWarning):
-        got = cudf.read_parquet(fpath)
-        fname = sorted(glob(fpath))[0]
-        expect = pd.read_parquet(fname)
-        expect = expect.apply(pd.to_numeric)
-        assert_eq(expect, got)
+def test_multifile_parquet_folder(tmpdir):
+
+    test_pdf1 = make_pdf(nrows=10, nvalids=10 // 2)
+    test_pdf2 = make_pdf(nrows=20)
+    expect = pd.concat([test_pdf1, test_pdf2])
+
+    sub_dir = tmpdir.mkdir("multi_part")
+    print(os.listdir(sub_dir))
+    print(os.listdir(tmpdir))
+    print(os.listdir(tmpdir.join("multi_part")))
```

Comment:
debug prints?

Reply:
Sorry about that. Missed the debug prints.